DOM+js+html/ui/script.js

```diff
@@ -0,0 +1,579 @@
+    new Photopost("6", "description6", new Date("2018-03-26T23:00:00"), "Vasia", "images/face.jpg", new Set(["Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("7", "description7", new Date("2018-01-26T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("8", "description8", new Date("2018-04-26T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("9", "description9", new Date("2018-02-22T23:00:00"), "Vasia", "images/face.jpg", new Set(["Dima"]), new Set(["#cool", "#2018"])),
+    new Photopost("10", "description10", new Date("2018-02-23T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("11", "description11", new Date("2018-02-21T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("12", "description12", new Date("2018-03-14T16:20:00"), "Vasia", "images/face.jpg", new Set(["Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("13", "description13", new Date("2018-02-19T23:00:00"), "Dima", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#soocool", "#2017"])),
+    new Photopost("14", "description14", new Date("2018-02-27T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#2017"])),
+    new Photopost("15", "description15", new Date("2018-03-14T16:20:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#soocool", "#2017"])),
+    new Photopost("16", "description16", new Date("2018-03-14T16:20:00"), "Vasia", "images/face.jpg", new Set(["Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("17", "description17", new Date("2018-02-17T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia"]), new Set(["#cool", "#2018"])),
+    new Photopost("18", "description18", new Date("2018-02-16T23:00:00"), "Vasia", "images/face.jpg", new Set(["Dima"]), new Set(["#cool", "#2018"])),
+    new Photopost("19", "description19", new Date("2018-02-15T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#cool", "#2018"])),
+    new Photopost("20", "description20", new Date("2018-02-14T23:00:00"), "Vasia", "images/face.jpg", new Set(["Vasia", "Petia"]), new Set(["#cool", "#2018"]))
+];
+let js = (function () {
+
+    function datesort(a, b) {
+        let dif = a.createdAt - b.createdAt;
+        if (dif > 0) {
+            return 1;
+        }
+        if (dif < 0) {
+            return -1;
+        }
+        return 0;
+    }
+
+    function getPhotoPosts(skip, top, filterConfig) {
+        let buffmass1 = [];
+        skip = skip || 0;
+        if (typeof(skip) !== "number") {
+            skip = 0;
+        }
+        top = top || 10;
+        if (typeof(top) !== "number") {
+            top = 0;
+        }
+        photoPosts.sort(datesort);
+        if (typeof(filterConfig) !== "undefined") {
```

Comment:
you copy code in ``if-else`` statement
you could also read about ``filter`` function in js